internal/apps/poll/restrict.go

```diff
@@ -72,8 +76,13 @@ func RestrictPoll(r restricter.HasPermer, canSee, canManage string, restrictedFi
 			return nil, fmt.Errorf("unmarshal poll state: %w", err)
 		}
 
-		if state != StatePublished && state != StateFinished {
```

Comment:
Please stick to this form: Explicitly name the states, which are allowed to see sensitive things.

Reply:
Done